backend/apps/cache/services/cache_manager.py

```
@@ -392,10 +394,12 @@ def invalidate_cache_on_save(sender, instance, created, **kwargs):
 try:
     manager = CacheManager()
     manager.invalidate_dependencies(instance)
-except Exception:
-    # Silently ignore errors during migrations/cache invalidation
-    pass
-        pass
+except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)